sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -664,7 +711,19 @@ def begin_recognize_custom_forms_from_url(self, model_id, form_url, **kwargs):
 
         if not model_id:
             raise ValueError("model_id cannot be None or empty.")
+        
+        pages = kwargs.pop("pages", None)
 
+        # FIXME: part of this code will be removed once autorest can handle diff mixin
+        # signatures across API versions
+        if pages:
+            if self._api_version == FormRecognizerApiVersion.V2_1_PREVIEW:
+                kwargs.update({"pages": pages})
+            else:
+                raise ValueError(
+                    "'pages' is only available for API version V2_1_PREVIEW and up"
+                )
+        
         polling_interval = kwargs.pop(
```

Comment:
actually sorry, just realized, can we make these same changes to the async FormRecognizerClient?

Reply:
Yes will update this in the async client. Thanks for catching that!

Reply:
Done